src/processors/modulation/scanner_vibrato/ScannerVibrato.cpp

```diff
@@ -131,25 +136,44 @@ void ScannerVibrato::processAudio (AudioBuffer<float>& buffer)
 
     if (inputsConnected.contains (AudioInput))
     {
+        const auto stereoMode = stereoParam->get();
+
         // generate mod mix arrays
-        auto** modMixData = modsMixBuffer.getArrayOfWritePointers();
-        FloatVectorOperations::add (modMixData[0], modOutBuffer.getReadPointer (0), 1.0f, numSamples);
-        FloatVectorOperations::multiply (modMixData[0], depthParam.getSmoothedBuffer(), numSamples); // this also multiplies the signal by 0.5
-        //        FloatVectorOperations::multiply (modMixData[0], depthParam.getSmoothedBuffer(), modOutBuffer.getReadPointer (0), numSamples); // this also multiplies the signal by 0.5
-        for (int i = ScannerVibratoWDF::numTaps - 1; i >= 0; --i)
-            tapMixTable[i].process (modMixData[0], modMixData[i], numSamples);
+        auto** modMixDataCh0 = modsMixBuffer[0].getArrayOfWritePointers();
+        FloatVectorOperations::add (modMixDataCh0[0], modOutBuffer.getReadPointer (0), 1.0f, numSamples);
+        FloatVectorOperations::multiply (modMixDataCh0[0], depthParam.getSmoothedBuffer(), numSamples); // this also multiplies the signal by 0.5
 
-        const auto& audioInBuffer = getInputBuffer (AudioInput);
-        const auto numChannels = audioInBuffer.getNumChannels();
-        audioOutBuffer.setSize (numChannels, numSamples, false, false, true);
+        auto** modMixDataCh1 = modsMixBuffer[1].getArrayOfWritePointers();
```

Comment:
I couldn't really think of a way to streamline this double declaration of the two modMixes